openslides/utils/auth.py

```diff
@@ -39,6 +39,41 @@ def has_perm(user: Optional[CollectionElement], perm: str) -> bool:
     return has_perm
 
 
+def in_some_groups(user: Optional[CollectionElement], groups: Union[List[int], List[Group]]) -> bool:
+    """
+    Checks that user is in at least one given group. Groups can be given as a list
+    of ids or group instances.
+
+    User can be a CollectionElement of a user or None.
+    """
+
+    if len(groups) > 0:
+        if isinstance(groups[0], Group):
```

Comment:
This looks like ducktyping: What if the first element is a group but the second not ... ?

Reply:
This is the thing with weak typed languages as python. I expect the user go give a valid input described by the `groups` parameter with this type: `Union[List[int], List[Group]]`. So either a list of integers or a list of groups.

Reply:
removed this now.